modules/apps/transfer/keeper/grpc_query.go

```diff
@@ -142,3 +157,15 @@ func (k Keeper) TotalEscrowForDenom(c context.Context, req *types.QueryTotalEscr
 		Amount: amount,
 	}, nil
 }
+
+func validategRPCRequest(portID, channelID string) error {
```

Comment:
This function is copy-pasted from here: https://github.com/cosmos/ibc-go/blob/1323a733d5b0d83478f1362ebc64cb1e42fa8caa/modules/core/04-channel/keeper/grpc_query.go#L641-L651

we could technically export that one instead but since it's internal logic I'm not sure that would be better. Open to change it in that case.

Reply:
Yeah it could be a nice clean-up after I complete the issue :)